common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -726,9 +719,9 @@ class NativeAuthMsalController : BaseNativeAuthController() {
             fullCacheRecord.accessToken != null && fullCacheRecord.accessToken.shouldRefresh()
         ) {
             if (!fullCacheRecord.accessToken.isExpired) {
-                setAcquireTokenResult(acquireTokenSilentResult, silentTokenCommandParameters, cacheRecords)
+                setAcquireTokenResult(acquireTokenSilentResult, parametersWithScopes, cacheRecords)
                 val refreshOnCommand =
-                    RefreshOnCommand(parameters, this.asControllerFactory(), PublicApiId.MSAL_REFRESH_ON)
+                    RefreshOnCommand(parametersWithScopes, this.asControllerFactory(), PublicApiId.MSAL_REFRESH_ON)
```

Comment:
Have you tested this? I'm asking because this controller factory is new, and I don't believe we've tested our NativeAuthController with it.

Reply:
yes, I tested by modifying the sample app and performing different calls to getAccessToken. What I was not able to test was the expiration of token.